tokio/src/task/join.rs

```diff
@@ -102,6 +102,9 @@ impl<T> Future for JoinHandle<T> {
     fn poll(mut self: Pin<&mut Self>, cx: &mut Context<'_>) -> Poll<Self::Output> {
         use std::mem::MaybeUninit;
 
+        // Keep track of task budget
+        ready!(crate::coop::poll_proceed(cx));
```

Comment:
Again, not sure if this needs a yield point?

Reply:
Same here as above (awaiting _many_ `JoinHandle`s).